app/templates/Thread/show.latte

```diff
@@ -4,15 +4,17 @@
 {block content}
     {*<h1 class="page-header">Detail diskuse</h1>*}
 
-    {if !$thread->event_id}<a n:href="Thread:default" class="btn btn-default">
-            <i class="glyphicon glyphicon-chevron-left mr10"></i> Nástěnka</a>
-    {else}<a n:href="Thread:eventThreads#event-$thread->event_id" class="btn btn-default">
-            <i class="glyphicon glyphicon-chevron-left mr10"></i> Diskuse k akcím</a>
+    {if !$thread->event_id}
+        <a n:href="Thread:default" class="btn btn-default">
+        <i class="glyphicon glyphicon-chevron-left mr10"></i> Nástěnka</a>
+    {else}
+        <h1>{$eventTitle}</h1>
+        <a n:href="Thread:eventThreads#event-$thread->event_id" class="btn btn-default">
+        <i class="glyphicon glyphicon-chevron-left mr10"></i> Diskuse k akcím</a>
         <a n:href="Event:show $thread->event_id" class="btn btn-default">Přejít na akci</a>
     {/if}
-    <a href="#addPostForm" class="btn btn-default" title="Přidat komentář">
+        <a href="#addPostForm" class="btn btn-default" title="Přidat komentář">
         <i class="glyphicon glyphicon-comment"></i></a>
```

Comment:
Vím, že to tady obecně moc systematické není, ale snažil bych se držet nějakou hierarchii. Tj. Uzavření tagu buď na stejném řádku
```
<a href=""><i class="glyphicon glyphicon-comment"></i></a>
```
nebo kolem vnitřku
```
<a href="#addPostForm" class="btn btn-default" title="Přidat komentář">
        <i class="glyphicon glyphicon-comment"></i>
</a>
```

(není nutné měnit, trousím jen myšlenky)